README.md

```diff
@@ -1,2 +1,27 @@
-# guilde-plateformes-propres
-Guides et modèles pour standardiser la qualité des projets
+# RTS Development Guidelines
+
+A guide for promoting best practices among RTS code repositories.
+
+## Introduction
+
+The software industry is constantly evolving. Our own practices should similarly evolve to disseminate knowledge throughout teams, share ownership, make it easier for newcomers to start and for seasoned developers to contribute to any project.
+
+## Guides
+
+The following guides are offered to help you improve your repository:
+
+- [Readme Guide](docs/guides/README_GUIDE.md): Write a great introductory text for your repository.
+- [Contributing Guide](docs/guides/CONTRIBUTING_GUIDE.md): Ensure smooth collaboration and contribution to your project.
+- [Code of Conduct Guide](docs/guides/CODE_OF_CONDUCT_GUIDE.md): Provide guidelines to ensure everyone can safely contribute.
+- [License Guide](docs/guides/LICENSE_GUIDE.md): Pick an appropriate license.
+
+> [!TIP]
+> GitHub [Community Standards](https://opensource.guide/) are a great source of information and best practices. You can check that your project follows these standards at any time by checking _Community Standards_ under the _Insights_ section.
+
+## Contributing
+
+We welcome contributions from everyone. To get started, please read our [Contribution Guidelines](docs/CONTRIBUTING.md). Feel free to open issues, submit pull requests, or share your ideas.
```

Comment:
We could add the appropriate links here, to help people get started:
```suggestion
We welcome contributions from everyone. To get started, please read our [Contribution Guidelines](docs/CONTRIBUTING.md). Feel free to [open issues](https://github.com/SRGSSR/guilde-plateformes-propres/issues/new), [submit pull requests](https://github.com/SRGSSR/guilde-plateformes-propres/compare), or share your ideas.
```

For the "share your ideas", we could enable "Discussions" on this repo, and link it to: https://github.com/SRGSSR/guilde-plateformes-propres/discussions